### PR TITLE
Remove setting of per-country related artefacts.

### PR DESCRIPTION
**Pull request by danielroseman**
This has never been used, and doesn't currently work due to the volume of data in the artefacts collection. Removing it simplifies the migration to content-store.This has never been used and complicates the migration to content-store.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -102,7 +102,7 @@ GEM
       state_machine
     hashie (3.2.0)
     hike (1.2.3)
-    htmlentities (4.3.1)
+    htmlentities (4.3.3)
```
Comment (alext) [inline Gemfile.lock]: Why this change?

**Comment by alext** [comment]
Other than the `Gemfile.lock` change, this all looks good.  Will make the migration a lot easier.

**Comment by alext** [comment]
It's also worth mentioning that this feature doesn't currently work anyway due to the volume of data in the artefacts collection.

**Comment by danielroseman** [comment]
Reverted lock file, updated commit comment and force pushed.
